pkgs/development/python-modules/mergedb/default.nix

```
@@ -0,0 +1,38 @@
+{ lib, pkgs, fetchPypi, setuptools, pyyaml, colorama, jinja2, setuptools-scm, git, python3Packages }:
+
+let
+  pname = "mergedb";
+  version = "0.1.1";
+in
+python3Packages.buildPythonPackage {
+  inherit pname version;
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2034c18dca23456c5b166b63d94300bcd8ec9f386e6cd639c2f66e141c0313f9";
+  };
+
+  nativeBuildInputs = [ git python3Packages.setuptools ];
+  propagatedBuildInputs = with python3Packages; [
+    setuptools
+    setuptools-scm
+    pyyaml
+    colorama
+    jinja2
+  ];
```

`setuptools` and its friends are usually not runtime dependencies.

```suggestion
  propagatedBuildInputs = [
    pyyaml
    colorama
    jinja2
  ];

```